### PR TITLE
Add Segment Definition Mixin for Unified Profile

### PR DESCRIPTION
**Pull request by HyveMynd**
Please link to the issue: #786

### CODE REVIEW

**Comment by prabhum2** [comment]
what functionalities is this PR going to introduce and who are all the parties to be involved in approving this?

**Comment by HyveMynd** [comment]
- Most of the approvers are internal adobe. Not all of the have github accounts but I have notified them of the changes. For the most part, the Unified Profile team of which I am a part of.
- New functionality is still in discussions, but for now we intend to have unified profile segment definitions available via XDM rather than keeping them behind profile apis. This will allow segments to be used as a business entity for others to take advantage of.

File: extensions/adobe/experience/profile/segmentdefinition-evaluationinfo.schema.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          "properties": {
+            "xdm:enabled": {
+              "title": "Enabled",
+              "type": "boolean",
+              "description": "Is Enabled"
+            }
+          }
+        },
+        "xdm:synchronous": {
+          "title": "Batch",
```
Comment (prabhum2) [inline extensions/adobe/experience/profile/segmentdefinition-evaluationinfo.schema.json]: why is the title same in all 3 properties

**Comment by HyveMynd** [comment]
@prabhum2 approvers are myself (monroy@adobe.com) and Craig (cmathis@adobe.com)

**Comment by cmathis** [comment]
I would include @rkpandey and @mohrahit to the required approver list.

**Comment by HyveMynd** [comment]
@cmathis this is meant to be purely experimental as we try various implementations and prospects. Is there any process I should take in that case?

**Comment by mjasrotia** [comment]
@HyveMynd , Do we still need this PR to be reviewed ?

**Pull request closed without merging**